operator/config/crd/bases/customize.core.cnrm.cloud.google.com_controllerresources.yaml

```diff
@@ -16,6 +16,107 @@ spec:
   scope: Cluster
   versions:
   - name: v1alpha1
+    schema:
```

Comment:
Nit (no need to fix, and certainly not in this PR): I like to either split out the generated files into a separate commit, or mark them as generated using the approach in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/864